pushover.sh

```diff
@@ -93,11 +96,28 @@ message="$*"
 if [ ! -x "${CURL}" ]; then
     echo "CURL is unset, empty, or does not point to curl executable. This script requires curl!" >&2
     exit 1
+else
+	if ! [ -z "$proxconf" ]
+	then
+		tpconf="$proxconf"
+
+		proxconf="$(echo $proxconf | cut -f 3 -d\/)"
+		proxconf="$(echo $proxconf | cut -f 1 -d\:)"
+
+		chkProxy=$(ping -c 1 $proxconf)
```

Comment:
If the user has configured a proxy address, you can safely assume that it is reachable: Curl has its own error reporting for the case when the URI can't be resolved or reached